web-components/src/[sandbox]/examples/combobox.ts

```diff
@@ -8,7 +8,11 @@ const dropdownValue: string[] = [];
 
 export const comboBoxTemplate = html`
   <h3>Default</h3>
-  <md-combobox .options=${comboBoxOptions} placeholder="Placeholder" .value=${[comboBoxOptions[5]]}></md-combobox>
+  <md-combobox
+    style="width: 220px; display: inline-block;"
+    .options=${comboBoxOptions}
+    placeholder="PlaceholderPlaceholderPlaceholderPlaceholderPlaceholder"
```

Comment:
it adds truncation - but not the tooltip/title, right?

Reply:
And title too. As I understood we need title for options, and truncate placeholder.

Reply:
Lena how come i don't see the title changes here?

Reply:
Sorry, it's my mistake, do not merge branch with title. Added